pkg/operator/ceph/object/controller_test.go

```diff
@@ -330,4 +330,28 @@ func TestCephObjectStoreController(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, cephv1.ConditionReady, objectStore.Status.Phase, objectStore)
 	logger.Info("PHASE 3 DONE")
+
+	// Test the functionality of verifyObjectUserCleanup
+	// function returns true if there are no cephobjectusers,
+	// otherwise false
+	logger.Info("STARTING PHASE 4")
+	_, userexists := r.verifyObjectUserCleanup(objectStore)
```

Comment:
The var `userexists` seems backwards to me. What about this?
```suggestion
	_, okToDelete := r.verifyObjectUserCleanup(objectStore)
```